### ISSUE TITLE
Logout Link Displayed Wrong

### ISSUE THREAD

**Issue by MLF2kx** [yiisoft/yii2#11488]
### What steps will reproduce the problem?
Login to site
### What is the expected result?
The logout link displayed as other links in navbar
### What do you get instead?
It's placed on top of navbar.

### Additional info
The logout form is currently made by this code:

```
'<li>'
. Html::beginForm(['/site/logout'], 'post')
. Html::submitButton(
'<span class="fa fa-sign-out fa-lg fa-fw"></span> خروج (' . Yii::$app->user->identity->name . ')',
['class' => 'btn btn-link']
)
. Html::endForm()
. '</li>'
```

But if we add 'navbar-form' class to the form, it will be displayed well:

```
'<li>'
. Html::beginForm(['/site/logout'], 'post', ['class' => 'navbar-form'])
. Html::submitButton(
'<span class="fa fa-sign-out fa-lg fa-fw"></span> خروج (' . Yii::$app->user->identity->name . ')',
['class' => 'btn btn-link']
)
. Html::endForm()
. '</li>'
```

Thanks for your review.

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.8
| PHP version      | 5.6.10
| Operating system | Mac OS X 10.11.4

**Comment by SilverFire** [comment]
I think it was fixed here: https://github.com/yiisoft/yii2-app-advanced/commit/6fdec1f5da00c456dd0cdd8c679066c760d523ae

**Comment by mmshlrfsh** [comment]
I'm using Basic template and it's not fixed there.

**Comment by SilverFire** [comment]
We are open for any better ideas. Could you suggest how to solve this problem using BS only classes?

**Comment by mmshlrfsh** [comment]
I've placed the needed change. Just assign "navbar-form" class to forms inside navbar.

**Comment by SilverFire** [comment]
Ok, I'll check it later

**Comment by cebe** [comment]
fixed by https://github.com/yiisoft/yii2-app-basic/commit/c97c8c605d7a8b6f8ed1646093f131ca80bd1444 and released in 2.0.8 already.
